jobs/bok-choy_coverage.rb

```diff
@@ -14,7 +14,7 @@
 
     # Retrieve the coverage info as JSON
 
-    coverage_uri = JENKINS_JOB_URL + "/SHARD=1,TEST_SUITE=bok-choy,label_exp=jenkins-worker/#{last_build_num}/cobertura/_default_/api/json?depth=2"
```

Comment:
I thought the last PR stopped showing bok-choy coverage. Do we still need this at all?